src/hooks/Circles/useInfinitePosts.tsx

```diff
@@ -57,7 +59,7 @@ export function useInfinitePosts({ circleId }: useInfinitePostsProps) {
     error: postsError,
     hasNextPage,
     fetchNextPage,
-  } = useInfiniteQuery('posts', queryPosts, {
+  } = useInfiniteQuery(`posts-${circleTile?.circleId}`, queryPosts, {
```

Comment:
I don't think it is a big deal but should we use `['posts', circleTile?.circleId]` instead? Not quite sure what the advantages could be but I think is is how RQ expects you to use dependencies.